### PR TITLE
Enable SMBIOS APL

### PR DESCRIPTION
**Pull request by elCaxper**
In order to get BIOS information from OS, SMBIOS is enabled for APL.

### CODE REVIEW

File: Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c [hunk 1]
```diff
@@ -730,6 +890,12 @@ BoardInit (
     if (PcdGetBool (PcdFramebufferInitEnabled)) {
       PciWrite8 (PCI_LIB_ADDRESS(SA_IGD_BUS, SA_IGD_DEV, SA_IGD_FUN_0, PCI_COMMAND_OFFSET), \
                  EFI_PCI_COMMAND_MEMORY_SPACE | EFI_PCI_COMMAND_BUS_MASTER);
+    }
+	//
```
Comment (mauricema) [inline Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c]: Can we fix the indent ?

File: Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c [hunk 2]
```diff
@@ -97,6 +100,163 @@ CreateNhltAcpiTable (
...
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_BIOS_INFORMATION,
+    3, "Unknown date");
+
+  //
+  // SMBIOS_TYPE_SYSTEM_INFORMATION
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_SYSTEM_INFORMATION,
+    1, "Intel Corporation");
+  if (PlatformId == PLATFORM_ID_OXH) {
+    AsciiSPrint (TempStrBuf, sizeof (TempStrBuf), "%a\0", "OXH Client Platform");
```
Comment (mauricema) [inline Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c]: Let us use full name:  Leaf Hill CRB, Oxbow Hill CRB, Juniper Hill CRB, UP Squared Board, Gordon Peak MRB, MinnowBoad 3

File: Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c [hunk 3]
```diff
@@ -97,6 +100,163 @@ CreateNhltAcpiTable (
...
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_SYSTEM_INFORMATION,
+    6, "ApolloLake Client System");
+
+  //
+  // SMBIOS_TYPE_BASEBOARD_INFORMATION
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_BASEBOARD_INFORMATION,
+    1, "Intel Corporation");
+  if (PlatformId == PLATFORM_ID_OXH) {
+    AsciiSPrint (TempStrBuf, sizeof (TempStrBuf), "%a\0", "OXH Platform");
```
Comment (mauricema) [inline Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c]: Maybe change ??? platform to  ???? board ?

File: Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c (hunk 2 above)
Comment (elCaxper) [reply Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c]: I have changed it but I don't know if it is what you expected.

**Pull request merged**